src/node-binance-api.ts

```diff
@@ -638,6 +645,42 @@ export default class Binance {
         }
     }
 
+    unarmorKey(a:string):number[] {
+        // eslint-disable-next-line no-useless-escape
+        const m = /-----BEGIN [^-]+-----\n([A-Za-z0-9+\/=\s]+)\n-----END [^-]+-----|begin-base64[^\n]+\n([A-Za-z0-9+\/=\s]+)====/.exec(a);
+        if (m) {
+            if (m[1]) {
+                a = m[1];
+            } else if (m[2]) {
+                a = m[2];
+            } else {
+                throw new Error("RegExp out of sync");
+            }
+        }
+        return base64.decode(a);
+    }
+
+    generateSignature(query: string) {
+        const APISECRET = this.Options.APISECRET || this.APISECRET;
```

Comment:
you don't need this `or` , `this.APISECRET` is enough

Reply:
but down below (before last line of this method) there is used `this.Options.APISECRET` so I was a bit confused, why it was also used in this same method, so i had decided to use both of them